source/css/sass/patterns/molecules/item-details.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../mixins/sizes";
+@import "../../mixins/typography";
+
+.item-details {
+  @include blg-spacing("block-end", "small", "margin");
+}
+
+.item-detail-tag__title {
+  @include label-content-typography();
+  @include padding(0, "block");
+}
+
+.item-detail-tag {
+  list-style: none;
+  @include blg-spacing("block-end", "small", "margin");
+  @include label-tag-typography();
+  @include padding(0, "inline-start");
+}
+
+.item-detail-tag__single-line {
```

Comment:
This would be a modifier (`--` rather than `__`), but isn't needed/used.

Reply:
This is the same in tag-list so will that need changing as well?

Reply:
https://github.com/libero/pattern-library/blob/843b7e306d3acea6bedf5421b160fef29f6bfdeb/source/css/sass/patterns/molecules/tag-list.scss#L20

There `tag-list` is the block, `list` the element, and `single-line` the element modifier.